q2_viromics/_utils.py

```diff
@@ -0,0 +1,123 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2024, Bokulich Lab.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+import os
+import subprocess
+from typing import List
+
+EXTERNAL_CMD_WARNING = (
+    "Running external command line application(s). "
+    "This may print messages to stdout and/or stderr.\n"
+    "The command(s) being run are below. These commands "
+    "cannot be manually re-run as they will depend on "
+    "temporary files that no longer exist."
+)
+
+
+def get_full_path(file_name):
+    """
+    Convert a file name or a relative path to its absolute path.
+
+    Args:
+    file_name (str): The name of the file or the relative path to the file.
+
+    Returns:
+    str: The absolute path of the file.
+    """
+
+    return os.path.abspath(file_name)
+
```

Comment:
Why is this here? You are not using this function. It looks like you copied it from another plugin. 
Also whats wrong with just using `os.path.abspath(file_name)` directly?